docs/pages/faq.mdx

```diff
@@ -58,13 +58,14 @@ Teleport nodes connected to proxy server (IoT) | 2,000* | 2x 2-4 vCPUs/8GB RAM |
 ## Which version of Teleport is supported?
 
 | Release       | Long Term Support | Release Date         | Supported Until      | Min tsh version  |
-| --------------|-----| -------------------- | -------------------- | -------------------- |
-| 5.0           | Yes | November 24th, 2020  | November 24th, 2021  | 3.x                  |
-| 4.4           | Yes | October 20th, 2020   | October 20th, 2021   | 3.x                  |
-| 4.3           | Yes | July 8th, 2020       | July 8th, 2021       | 3.x                  |
-| 4.2 (EOL)     | Yes | December 19th, 2019  | December 19th, 2020  | 3.x                  |
-| 4.1 (EOL)     | Yes | October 1st, 2019    | October 1st, 2020    | 3.x                  |
-| 4.0 (EOL)     | Yes | June 18th, 2019      | June 18th, 2020      | 3.x                  |
+| --------------|-------------------| -------------------- | -------------------- | -------------------- |
+| 6             | Yes               | March 3rd, 2021      | March 3rd, 2022      | 3.x                  |
```

Comment:
I doubt that tsh version 3 will work with new cluster

Reply:
Just checked and `tsh` 3.1 and above continue to work with Teleport 6 for login and SSH.

That being said, I agree that we should provide a better answer to this question and what/if functionality will break by being too far incompatible.